package.json

```diff
@@ -10,9 +10,12 @@
     "test": "sk test",
     "lint": "sk lint",
     "type-check": "sk type-check",
-    "build": "sk build",
+    "build": "sk build; yarn run build:argo",
     "version-bump": "lerna version --no-push --include-merged-tags",
-    "deploy": "lerna publish from-package --yes"
+    "deploy": "lerna publish from-package --yes",
+    "build:argo": "lerna run build",
+    "server:argo": "yarn workspace @shopify/argo-admin-playground run server",
+    "simulator:argo": "yarn workspace @shopify/argo-admin-cli run server"
```

Comment:
If these commands stay, we really need to make it a lot more clear about what parts of this are admin-only.

Reply:
Sorry, I renamed them now. I'm not sure why I put the `argo` name in there. It was supposed to say `admin`